package.json

```diff
@@ -4,7 +4,7 @@
 	"author": "Tobias Koppers @sokra",
 	"description": "Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jade, coffee, css, less, ... and your custom stuff.",
 	"dependencies": {
-		"esprima": "~1.2.0",
+		"esprima-fb": "^8001.2001.0-dev-harmony-fb",
```

Comment:
I *think* I saw NPM 2.0 erroring on this kind of thing since it's not allowed to reference `-dev` versions in new semver. Not sure though.